docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/crowdstrike-threat-intel-source.md

```diff
@@ -0,0 +1,127 @@
+
+### Vendor configuration
+
+**Prerequisites**
+
+To collect vulnerability data from the CrowdStrike platform, you must have an authorized CrowdStrike account. CrowdStrike APIs use an OAuth 2.0 authorization token to make authorized API calls. CrowdStrike API Client is required to get the OAuth 2.0 authorization token. To define a CrowdStrike API client, you must be designated as a `CrowdStrike Falcon Administrator` role.
+
+In this configuration, you will set up the CrowdStrike Threat Intel and configure it to be authorized and authenticated to use indicator information from CrowdStrike Threat Intel API. To obtain the auth token, you will need the following parameters.
+
+#### API Client and API Secret
+
+The **API security token** is used to authenticate with CrowdStrike Threat Intel API. After successfully creating the API client, you will get the **Client Id**, **Client Secret**, and **Base URL**.
+
+To get the **CrowdStrike API Client**, follow the steps below:
+
+1. Sign in to [CrowdStrike Platform](https://falcon.crowdstrike.com/login/).
+1. Sign in using your email address and password. After you've completed the two-factor authentication, you'll be directed to the application dashboard.<br/><img src={useBaseUrl('img/send-data/crowdstrike-fdr-host-inventory-login-screen.png')} style={{border: '1px solid gray'}} alt="crowdstrike-spotlight-login-screen.png" width="600" />
+1. From the CrowdStrike console, select the **Support and resources** option from the menu.<br/><img src={useBaseUrl('img/send-data/crowdstrike-fdr-host-inventory-support-and-resources.png')} style={{border: '1px solid gray'}} alt="<crowdstrike-fspotlight-support-and-resources.png>" width="600" />
+1. In the **Resources and tools** section, select the **API clients and keys** option. You can then view existing clients or add new API clients from there.<br/><img src={useBaseUrl('img/send-data/crowdstrike-fdr-host-inventory-api-key-client.png')} style={{border: '1px solid gray'}} alt="<crowdstrike-spotlight-api-key-client.png>" width="600" />
+1. Click **Add new API client**. You will be prompted to give a descriptive name and select the appropriate API scopes.<br/><img src={useBaseUrl('img/send-data/crowdstrike-fdr-host-inventory-new-client.png')} style={{border: '1px solid gray'}} alt="<crowdstrike-spotlight-new-client.png>" width="600" />
+1. Provide a proper name and description and select the **indicators(Falcon Intelligence):read scope** scope. Click on `ADD` to complete the process.
+1. After you click on `ADD` a dialogue box will appear with the **Client ID**, **Client Secret**, and **Base URL**. Copy and save the Client Id, Client Secret, and Base URL to a folder location because you will need them while configuring the [CrowdStrike Threat Intel Source](#source-configuration).<br/><img src={useBaseUrl('img/send-data/crowdstrike-fdr-host-inventory-copy-creds.png')} style={{border: '1px solid gray'}} alt="<crowdstrike-spotlight-copy-creds.png>" width="600" />
```

Comment:
```suggestion
1. After you click on `ADD` a dialogue box will appear with the **Client ID**, **Client Secret**, and **Base URL**. Copy and save the Client Id, Client Secret, and Base URL to a folder location because you will need them for the [source configuration](#source-configuration).<br/><img src={useBaseUrl('img/send-data/crowdstrike-fdr-host-inventory-copy-creds.png')} style={{border: '1px solid gray'}} alt="<crowdstrike-spotlight-copy-creds.png>" width="600" />
```